package-lock.json

```diff
@@ -9,7 +9,8 @@
       "version": "0.0.1",
       "license": "MIT",
       "dependencies": {
-        "@octokit/rest": "^21.0.2"
+        "@octokit/app": "^15.1.0",
```

Comment:
> "GitHub encourages the use of the GitHub App workflow when a system needs
> to interact with the GitHub API on its own behalf, rather than using
> personal access tokens which need to be regenerated on a regular basis."  
It would be helpful to link to this guidance in the commit message for future reference

Reply:
Done.